src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -113,6 +167,17 @@ const BarChartCard = ({
 
   const scaleType = series && series.timeDataSourceId ? 'time' : 'labels';
 
+  const axes = mapValuesToAxes(series, layout);
+
+  // Set the colors for each dataset
+  const colors = { identifier: 'group', scale: {} };
```

Comment:
would be good to export this and write a testcase around it

Reply:
Added